internal/testrunner/runners/system/runner.go

```diff
@@ -373,26 +373,26 @@ func createPackageDatastream(
 	r.Package.Title = pkg.Title
 	r.Package.Version = pkg.Version
 
-	r.Inputs = []ingestmanager.Input{
+	r.Inputs = []kibana.Input{
 		{
 			Type:    streamInput,
 			Enabled: true,
 		},
 	}
 
-	streams := []ingestmanager.Stream{
+	streams := []kibana.Stream{
 		{
 			ID:      fmt.Sprintf("%s-%s.%s", streamInput, pkg.Name, ds.Name),
 			Enabled: true,
-			DataStream: ingestmanager.DataStream{
+			DataStream: kibana.DataStream{
 				Type:    ds.Type,
 				Dataset: fmt.Sprintf("%s.%s", pkg.Name, ds.Name),
 			},
 		},
 	}
 
 	// Add dataStream-level vars
-	dsVars := ingestmanager.Vars{}
+	dsVars := kibana.Vars{}
```

Comment:
Thinking loud (nothing to be done immediately): The `Vars` structure may look to be a generic one. It may happen in the future that we need to handle another `Vars` structure, so some of these structs will require a relevant prefix added to their names.